src/pages/wookchanglee/Detail/Reviews/Reviews.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Review from '../Review/Review';
+import './Reviews.scss';
+
+class Reviews extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      review: [],
+      tempUsername: '',
+      tempReviewContent: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/reviewListData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ review: data });
+      });
+  }
+
+  usernameRef = React.createRef();
+  contentRef = React.createRef();
+
+  writeReview = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value });
+  };
+
+  deleteReview = id => {
+    let reviewArr = [...this.state.review];
+    reviewArr = reviewArr.filter(el => el.id !== id);
+    this.setState({ review: reviewArr });
+  };
+
+  submitReview = e => {
+    if (e.key === 'Enter') {
+      if (
+        this.state.tempUsername.trim() !== '' &&
+        this.state.tempReviewContent.trim() !== ''
+      ) {
+        const reviewArr = [...this.state.review];
+        const newId =
+          reviewArr.length > 0 ? reviewArr[reviewArr.length - 1].id + 1 : 1;
+        reviewArr.push({
+          username: this.state.tempUsername,
+          reviewContent: this.state.tempReviewContent,
+          id: newId,
+        });
+        this.setState({ review: reviewArr });
+        this.usernameRef.current.value = '';
+        this.contentRef.current.value = '';
+        this.setState({
+          tempReviewContent: '',
+          tempUsername: '',
+        });
```

Comment:
```suggestion
        this.setState({
          tempReviewContent: '',
          tempUsername: '',
        });
```
createRef를 사용하지 않고 구현하면 이렇게 할 수 있습니다 :)

Reply:
input의 value에 tempUsrname과 tempReviewContent를 넣어 ref를 없앴습니다 감사합니다!!!